### PR TITLE
Added the delegate method scrollViewDidEndDecelerating: to TUIScrollView

### PR DESCRIPTION
**Pull request by jagreenwood**
If implemented by the delegate, the scroll view calls this method when the scrolling movement comes to a halt.  Functionality mimics that of scrollViewDidEndDecelerating: in UIKit.

### CODE REVIEW

File: lib/UIKit/TUIScrollView.m [hunk 1]
```diff
@@ -772,6 +773,9 @@ - (void)_updateBounce
 		
 		if(fabsf(_bounce.vy) < 1.0 && fabsf(_bounce.y) < 1.0 && fabsf(_bounce.vx) < 1.0 && fabsf(_bounce.x) < 1.0) {
 			[self _stopTimer];
+            if (_scrollViewFlags.delegateScrollViewDidEndDecelerating) {
```
Comment (jwilling) [inline lib/UIKit/TUIScrollView.m]: Can you check your formatting here, and below? It looks like you have two tabs extra padding on both.

**Comment by jagreenwood** [comment]
Fixed formatting caused by bad Xcode settings.

**Comment by jwilling** [comment]
Hmm, still looks quite off. Can you check the commit preview here on GH and see how it looks? You'll see what I mean.

**Comment by jagreenwood** [comment]
Ugh, I'll kill the PR and redo it so as not to be all commit spammy

**Pull request closed without merging**